server/sql/migrations/00005_add_and_alter_columns_to_users_and_subscriptions.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE users ADD COLUMN user_uuid UUID NOT NULL DEFAULT gen_random_uuid();
+
+ALTER TABLE subscriptions ADD COLUMN subscription_uuid UUID NOT NULL DEFAULT gen_random_uuid();
+
+ALTER TABLE subscriptions ALTER COLUMN amount TYPE money;
```

Comment:
we can do this, but i'm not sure the `money` type will be best. i know previously we've ran into a lot of issues when dealing with currency that isn't in cents (just an `integer`) but... perhaps we can navigate around this

Reply:
I changed the column 'amount' to type 'decimal'.